lib/vendor/noitidart-ostypes/cutils.jsm

```diff
@@ -0,0 +1,462 @@
+/*** CTypes Utilities ***
+ * Stuff that I find useful in my CTypes work under all Operating Systems
```

Comment:
Alright, ignore me. There isn't a package.json file in that upstream repo, so npm chokes pretty hard. I think before we were resorting to using bower to install some of these non-npm GitHub modules.

FWIW, I filed https://github.com/Noitidart/ostypes/issues/9. We could possibly even craft up a bare bones package.json and submit a PR and see if that is easier to get merged quickly, if we do want to go the npm dependency route.